mscore/inspector/inspector.cpp

```diff
@@ -160,13 +160,10 @@ void Inspector::setElements(const QList<Element*>& l)
 
             bool sameTypes = true;
             for (Element* ee : _el) {
-                  if (_element->type() != ee->type())
+                  if ((_element->type() != ee->type()) && // different and
+                      (!_element->isSlurTieSegment() && !ee->isSlurTieSegment()) || // neither Slur nor Tie either side, or
+                      (ee->isNote() && toNote(ee)->chord()->isGrace() != toNote(_element)->chord()->isGrace())) // HACK
```

Comment:
Thanks for the clarification.

It is `if (A && B || C)`, so if A is true, B is evaluated, If a or B is false, C is evaluated, should be no change to the previous `else if`, right? Or do we need some extra brackets?